### ISSUE TITLE
After new Spyder IDE installation, clicked on popup to read "tutorial"

### ISSUE THREAD

**Issue by medlaw** [spyder-ide/spyder#13422]
### What steps will reproduce the problem?

Clicked on popup: "New to Spyder? Read our tutorial" after initial installation

### Traceback
```python-traceback
Traceback (most recent call last):
  File "C:\Users\Admin\Downloads\spyder-master\spyder\widgets\browser.py", line 335, in contextMenuEvent
    self.context_menu.popup(event.globalPos())
AttributeError: 'NoneType' object has no attribute 'popup'
```

## Versions

* Spyder version: 5.0.0.dev0 
* Python version: 3.8.4
* Qt version: 5.12.9
* PyQt5 version: 5.12.3
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0        :  1.4.0 (OK)
chardet >=2.0.0             :  3.0.4 (OK)
cloudpickle >=0.5.0         :  1.5.0 (OK)
cookiecutter >=1.6.0        :  None (OK)
diff_match_patch >=20181111 :  20200713 (OK)
intervaltree                :  None (OK)
IPython >=4.0               :  7.16.1 (OK)
jedi =0.17.1                :  0.17.1 (OK)
nbconvert >=4.0             :  5.6.1 (OK)
numpydoc >=0.6.0            :  1.1.0 (OK)
paramiko >=2.4.0            :  2.7.1 (OK)
parso =0.7.0                :  0.7.0 (OK)
pexpect >=4.4.0             :  4.8.0 (OK)
pickleshare >=0.4           :  0.7.5 (OK)
psutil >=5.3                :  5.7.2 (OK)
pygments >=2.0              :  2.6.1 (OK)
pylint >=1.0                :  2.5.3 (OK)
pyls >=0.34.0;<1.0.0        :  0+unknown (OK)
qdarkstyle >=2.8            :  2.8.1 (OK)
qtawesome >=0.5.7           :  0.7.2 (OK)
qtconsole >=4.6.0           :  4.7.5 (OK)
qtpy >=1.5.0                :  1.9.0 (OK)
sphinx >=0.6.6              :  3.1.2 (OK)
spyder_kernels >=2.0.0dev0  :  2.0.0.dev0 (OK)
watchdog                    :  None (OK)
zmq >=17                    :  19.0.1 (OK)

# Optional:
cython >=0.21               :  None (OK)
matplotlib >=2.0.0          :  3.3.0 (OK)
numpy >=1.7                 :  1.19.1 (OK)
pandas >=0.13.1             :  1.1.0 (OK)
scipy >=0.17.0              :  1.5.2 (OK)
sympy >=0.7.3               :  None (OK)
```

**Comment by goanpeca** [comment]
Will fix @dalthviz

**Comment by dalthviz** [comment]
Thanks @goanpeca :+1:

**Comment by medlaw** [comment]
Thank you for your kind, valuable and expert attention.

The app was installed via pip -- and was run via bootstrap.py

I have noted that if run via an "elevated" command line as python bootstrap.py the app will run apparently flawlessly -- including access to the tutorial via "help" -- AND -- kite completion also will run

I further noted that when a "non-elevated" command line is used, "protection" errors occur and the app will fail to run

I still further noted that I can run the app successfully following generation of a desktop link via a "send to desktop" -- where my default app to run .py files is python 3.8

Following that, I discovered that changing the desktop shortcut icon mysteriously caused the app to fail

In summary, the app will run -- apparently flawlessly -- if bootstrap.py is launched via an elevated command line environment -- which ostensibly avoids fatal "protection" errors occasioned by the script -- and will fail to run via a "non-elevated" command line environment

Thank you again.
